cost/budget_alerts/budget_alert.pt

```diff
@@ -346,24 +453,27 @@ escalation "esc_budget_alert" do
 end
 
 policy "budget_alert" do
-  validate_each $filtered_target do
+  validate $filtered_target do
     summary_template "Budget Exceeded"
     # ignore changes to the monthly spend and runrate, since those will change constantly
     hash_exclude "actual", "runrate", "total"
     detail_template <<-EOS
 # Budget Exceeded
 
 ### Cost Metric: {{ parameters.param_cost_metric }}
-### Budget Alert Type: {{ parameters.param_type }}
+### Budget Alert Type: {{ parameters.param_type }} Budget Exceeded
```

Comment:
remove budget exceeded here.  the param_type should be in the summary_template above.

Reply:
I have addressed this. Please review